types/kurento-client/index.d.ts

```diff
@@ -91,6 +91,8 @@ declare namespace kurento {
         ): Promise<PlayerEndpoint>;
         create(type: string, options?: Record<string, unknown>): Promise<MediaElement>;
         getMediaobjectById(objectId: string): Promise<MediaPipeline | WebRtcEndpoint | RecorderEndpoint>;
+        // tslint:disable-next-line
+        getMediaobjectById<T extends MediaPipeline | WebRtcEndpoint | RecorderEndpoint>(objectId: string): Promise<T>;
```

Comment:
```suggestion
        getMediaobjectById<T extends MediaObject = MediaObject>(objectId: string): Promise<T>;
```

How is this change, because `MediaPipeline`, `WebRtcEndpoint`, and `RecorderEndpoint` all extend `MediaObject`?

In addition, with this change, you can remove the line above (94th line) because this sets the default type for `getMediaobjectById` to be used without specifying the type.

Reply:
Nice, yeh this is better 👍